### DISCUSSION TITLE
Row freezing/multiline headers

### DISCUSSION THREAD

**Discussion by MrFahrenheit-55** [bosskmk/pluto_grid#643] [Ideas]
Hi, would it be possible to look at possible row freezing or multiline headers? I use Pluto grid as a Pivot table and sometimes need more than a single line header for data.

Thanks for the great widget!

**Comment by bosskmk** [comment]
https://weblaze.dev/pluto_grid/build/web/#feature/column-group

You can express the title of the column with the column group function in the link above or `PlutoColumn.titleSpan`.
However, no pivot feature is provided.

**Reply by MrFahrenheit-55** [reply]
Thanks for the reply @bosskmk, that group function did the trick. 
Don't worry about the pivot feature, the data is formatted to look like a pivot.

**Reply by bosskmk** [reply]
Yep, I'd like to add a pivot function someday.
Glad the above link solved your problem.
